test/x/track_test.rb

```diff
@@ -4,11 +4,8 @@
 module X
   class TrackTest < Minitest::Test
     def test_tracks
-      f= './test/fixtures/xapi_v3_tracks.json'
```

Comment:
I tried doing this and there are some tests that were failing , e.g. `user_track_test.rb`. I'll dig deeper and rewrite those tests since they assume a different fixture. I'll need a bit of guidance though since I noticed that those tests were pretty long.